packages/js/product-editor/src/blocks/generic/checkbox/edit.tsx

```diff
@@ -39,7 +39,7 @@ export function Edit( {
 	return (
 		<div { ...blockProps }>
 			<Checkbox
-				value={ value || null }
+				value={ value || false }
```

Comment:
I changed this because when `value = false`, we used to send `null` to the `Checkbox` component. That was problematic on multiple levels.